pkgs/development/tools/continuous-integration/woodpecker/woodpecker-package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "start": "vite",
-    "build": "vite build",
+    "build": "vite build --base=/BASE_PATH",
```

Comment:
I foresee an issue here, I'll try building this at some point today.

Reply:
I was able to build and update my server without issues.

Reply:
Ah I had misunderstood what that option meant (I thought it did something on the file-system). Looking at the PR that introduced it, I see that it's part of their support for sub-folder proxying.